project-1/euler-problem-1.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/* Problem description:
+If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9.
+The sum of these multiples is 23.
+
+Find the sum of all the multiples of 3 or 5 below 1000.
+*/
+//not trying to do any fancy tdd or function building right off here - just want to solve the problem fist =]
```

Comment:
Since `$sum_of_multiples` is null, if we add to it, it'll get a value, but let's start that puppy off at a defined 0 first. Like this:
```
$sum_of_multiples = 0;
```